src/test/util/setup_common.cpp

```diff
@@ -91,6 +93,16 @@ void Seed(FastRandomContext& ctx)
     ctx = FastRandomContext(seed);
 }
 
+CAmount SemiRandMoneyAmount()
```

Comment:
No thoughts on the change, but not sure this code should be in `setup_common`. Probably `src/test/util/something` ?

Reply:
Done. Didn't see a good existing home for it, so I moved the random generation test code from `setup_common` to `src/test/util/random`, which seems beneficial as many of the unit tests don't call it, and then added the test helper there.